orchestrator/README.md

```diff
@@ -28,6 +28,26 @@ Use the following environment variables to configure the orchestrator:
 - `ORCA_CAREPLANCONTRIBUTOR_FRONTEND_URL`: Base URL of the frontend application, to which the browser is redirected on app launch (default: `/frontend/enrollment`).
 - `ORCA_CAREPLANCONTRIBUTOR_SESSIONTIMEOUT`: Configure the user session timeout, use Golang time.Duration format (default: 15m).
 
+### Token Client configuration
+- `ORCA_CAREPLANCONTRIBUTOR_TOKENCLIENT_ENABLED`: Enable the Token Client (default: `false`).
+- `ORCA_CAREPLANCONTRIBUTOR_TOKENCLIENT_CLIENTID`: ClientID used for token validation, this will typically be the same as the `aud` claim in the JWT being validated.
+The following two fields can be repeated for multiple trusted issuers, but must both be set for each issuer:
+- `ORCA_CAREPLANCONTRIBUTOR_TOKENCLIENT_<ISSUER_NAME>_ISSUERURL`: Same as the `iss` claim in the JWT.
+- `ORCA_CAREPLANCONTRIBUTOR_TOKENCLIENT_<ISSUER_NAME>_DISCOVERYURL`: Same as the `iss` claim in the JWT, but with `tfp` before `/v2.0/` as well as the `.well-known/openid_configuration` suffix.
```

Comment:
it seems `tfp` is Microsoft-specific. Maybe good to note in this documentation that it has been tested with MS, but should work with other OpenID COnnect Providers?